content/blog/zephyr-rtos-on-esp32-first-steps/index.md

```diff
@@ -44,38 +44,37 @@ sudo apt upgrade
 
 Run __apt__  to install dependencies:
 
-```
-sudo apt install --no-install-recommends git cmake \
-ninja-build gperf ccache dfu-util device-tree-compiler wget \
-python3-dev python3-pip python3-setuptools python3-tk \
-python3-wheel xz-utils file make gcc gcc-multilib \
-g++-multilib libsdl2-dev
+```sh
+sudo apt install --no-install-recommends \
+git cmake ninja-build gperf ccache dfu-util device-tree-compiler wget \
+python3-dev python3-pip python3-setuptools python3-tk python3-wheel \
+xz-utils file make gcc gcc-multilib g++-multilib libsdl2-dev libmagic1
 ```
 
 Check the __CMake__  version on your host.
 
-```
+```sh
 cmake --version
 ```
 
 If the version displayed is __3.13.1__  or above then skip to the next section. Otherwise, follow these three steps to update CMake:
 
 Add the Kitware signing key:
 
-```
+```sh
 wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | sudo apt-key add -
 ```
 
 Add the Kitware apt repository for your OS release. For Ubuntu 20.04 LTS:
```

Comment:
recent Zephyr guideline states the Kitware is necessary for Ubuntu 22.04 and older.